### PR TITLE
Instructions review

### PR DESCRIPTION
**Pull request by lucie-docs**
Hi @hectorsector, here are the new and improved instructions for this lab for your review :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,54 @@
...
+**Quick links:**
+- [About this lab](#about-this-lab)
+- [Instructions](#instructions)
+- [Seeing your result](#seeing-your-result)
+- [Troubleshooting](#troubleshooting)
+- [Useful resources](#useful-resources)
+
+## About this lab
+
+:warning: A grading script and a setup utility exist inside of the `.github/` directory. You do not need to use these workflows for any purpose and **altering their contents will affect the repository's ability to assess your lab and give feedback.**
```
Comment (hectorsector) [inline README.md:14]: We may want to explicitly call out `.github/workflows/grading.yml` here, as the other workflows inside `.github` are the ones the learner will engage with.

File: README.md [hunk 2]
```diff
@@ -1,41 +1,54 @@
...
+
 In the codebase for this repo, you'll find 3 workflows which we describe below with their intended purpose. Each workflow is complete **except for** the `on:` key.
-1. `stale-daily.yaml`: checks for stale issues daily at midnight and closes them
-2. `stale-weekly.yaml`: checks for stale issues every Monday and closes them
-3. `stale-monthly.yaml`: checks for stale issues on the first of every month and closes them
 
-A grading script exists at `.github/grading.yml`. You do not need to use this workflows for any purpose and altering its contents will affect the repository's ability to assess your lab and give feedback.
+- `stale-daily.yaml`: checks for stale issues daily at midnight and closes them
+- `stale-weekly.yaml`: checks for stale issues every Monday and closes them
+- `stale-monthly.yaml`: checks for stale issues on the first of every month and closes them
```
Comment (hectorsector) [inline README.md:26]: Similarly, we can probably make this more clear by using full paths: `.github/workflows/sale-daily.yml`. Alternatively, we can just mention above this section that these files are all in `.github/workflows/`

I also made a mistake in these file names! Extensions are all `.yml` not `.yaml`

File: README.md [hunk 3]
```diff
@@ -1,41 +1,54 @@
...
+
+Please complete the instructions below:
+1. Create your own copy of this repository by using the [Use this template](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template#creating-a-repository-from-a-template) button.
+2. In your newly created repository, create a new branch.
+3. On your new branch, edit the `on` key of each `stale` workflow so that:
+    - `stale-daily.yaml` contains a trigger to run daily at midnight
+    - `stale-weekly.yaml` contains a trigger to run every Monday
+    - `stale-monthly.yaml` contains a trigger to run on the first of every month
```
Comment (hectorsector) [inline README.md:34]: Same thing here, I think we want to reference their full path and replace extensions with `.yml`.

**Review by hectorsector — CHANGES_REQUESTED** [review]
Thanks @lucie-docs! Some notes -- mainly about mistakes I made. Let me know if you'd like me to contribute any of these changes.

File: README.md (hunk 1 above)
Comment (lucie-docs) [reply README.md:14]: Agreed!

File: README.md (hunk 2 above)
Comment (lucie-docs) [reply README.md:26]: Ok, will correct!

**Review by hectorsector — APPROVED** [review]
:shipit: Thanks @lucie-docs let's merge!

**Pull request merged**